tests/test_create_pod_attaching_multiple_pvc_start_IO.py

```diff
@@ -0,0 +1,376 @@
+    rbd_pvc_data['rbd_pvc_name'] = volName
+    log.info(f"rbd_pvc_data {rbd_pvc_data}")
+    template = os.path.join(TEMPLATES_DIR, "pvc.yaml")
+    log.info(f"Create Persistent Volume Claim ")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template, **rbd_pvc_data
+    )
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    assert PVC.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    # Check for pv creation succeed from oc
+    try:
+        PVC.wait_for_resource(
+            resource_name=volName, condition='Bound', timeout=20
+        )
+        log.info(f"PVC {volName} creation success")
+        return True
+    except exceptions.CommandFailed:
+        log.info(f"Exception PVC {volName} not Bound")
+        return False
+
+
+def create_pod(pv_list):
+    """
+    Generate cirros yaml with multiple pvc and run IO's
+
+    """
+    yaml_data = os.path.join(CIRROS_DIR, "cirros.yaml")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        yaml_data
+    )
+    cirros_pod_name = data['metadata']['name']
+    temp_dict1 = []
+    temp_dict2 = []
+    temp_str = ''
+    temp_list1 = data['spec']['template']['spec']['containers'][0]['args']
+    temp_list1.pop(len(temp_list1) - 1)
+    temp_list2 = data['spec']['template']['spec']['containers'][0][
+        'livenessProbe']['exec']['command']
+    temp_list2.pop(len(temp_list2) - 1)
+    for i in pv_list:
+        temp_dict1.append(
+            {'name': i, 'persistentVolumeClaim': {'claimName': i}}
+        )
+        temp_dict2.append({'mountPath': '/mnt' + i, 'name': i})
+        temp_list2.append(
+            'mount | grep /mnt' + i + ' && head -c 1024 '
+            '< /dev/urandom >> /mnt' + i + '/random-data.log'
+        )
+        temp_str = temp_str + '(mount | grep /mnt' + i + ') && (head -c ' \
+            '1000 < /dev/urandom > /mnt' + i + '/random-data.log) || exit 1;'
+    del data['spec']['template']['spec']['volumes']
+    del data['spec']['template']['spec']['containers'][0]['volumeMounts']
+    del data['spec']['template']['spec']['containers'][0][
+        'livenessProbe']['exec']['command']
+    data['spec']['template']['spec']['volumes'] = temp_dict1
+    data['spec']['template']['spec']['containers'][0][
+        'volumeMounts'] = temp_dict2
+    temp_list1.append('while true; do ' + temp_str + ' sleep 20 ; done')
+    data['spec']['template']['spec']['containers'][0][
+        'livenessProbe']['exec']['command'] = temp_list2
+    with open(TEMP_YAML, 'w') as yaml_data:
+        yaml.dump(data, yaml_data, default_flow_style=False)
+    log.info(f"Create cirros pod with all the created pvcs")
+    assert DC.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    try:
+        # Check for pod is up and running
+        POD.wait_for_resource(
+            condition='Running', selector='deploymentconfig=cirrospod',
+            timeout=180, sleep=20
```

Comment:
please create cnst parameters at the beginning of this file with timeout and sleep.
As a general note, never use hardcoded in the code

Reply:
ack